ad/movement/ad-cs/certificate-templates.md

```diff
@@ -20,34 +20,84 @@ Following this, [Olivier Lyak](https://twitter.com/ly4k\_) has found two new tem
 
 ### Certificate mapping
```

Comment:
I'm not sure I understand the various types of mapping
I see implicit, explicit, and strong
Do you mind clarifying this a little bit more?

Reply:
I have added little explains to highlight the differences and the links between explicit/implicit and weak/strong mappings. I hope it is more clear ?